lib/src/airship_in_app.dart

```diff
@@ -1,18 +1,31 @@
-// class AirshipInApp {
-//
-//   const MethodChannel _channel;
-//
-//   const AirshipInApp._internal(this._channel)
-//
-//
-//   /// Pauses or unpauses in-app automation.
-//   static Future<void> setInAppAutomationPaused(bool paused) async {
-//     return await _channel.invokeMethod('setInAppAutomationPaused', paused);
-//   }
-//
-//   /// Checks if in-app automation is paused or not.
-//   static Future<void> get getInAppAutomationPaused async {
-//     return await _channel.invokeMethod('getInAppAutomationPaused');
-//   }
-//
-// }
\ No newline at end of file
+import 'airship_module.dart';
+
+class AirshipInApp {
+
+  final AirshipModule _module;
+
+  AirshipInApp(AirshipModule module) : this._module = module;
+
+  /// Pauses or unpauses in-app automation.
+  Future<void> setInAppAutomationPaused(bool paused) async {
```

Comment:
could be just `setPaused`, `isPaused` since we are already in the `AirshipInApp` module